src/image-cdn.js

```diff
@@ -1,6 +1,17 @@
-function imageUrlForId(id, size = 400) {
-  size = size == "full" ? "full/" : size > 400 ? "800/" : "";
-  return `https://${id % 7}.api.artsmia.org/${size}${id}.jpg`;
+function imageForCollections(art, size = 800) {
```

Comment:
We should align the function name here with the name we use in `require`/`import` statements, so in the spirit of touching less code, you might rename this function `imageCDN`. A JSDoc would be nice to clarify argument types:

```
/**
 * Get an image URL for a piece of art
 *
 * @param {object} art
 * @param {'full' | 400 | 800} size
 */
```